### ISSUE TITLE
Torrent Sections

### ISSUE THREAD

**Issue by JarHeadDev** [taobataoma/meanTorrent#10]
Notice script is current demoing just movies and tv series. Could this script be used solely for other content like tutorials and ebooks or would it need to be updated to support that type of content also?

Have to say the demo is very responsive. We're still setting up demo site to see what we can suggest to help the project.

Would be nice if in the future there was a way to connect the Donation page to WHMCS also and have WHMCS handle all the donations.

**Comment by taobataoma** [comment]
@JarHeadDev yes, this project will support music and other type future, if the type is too more, it can be set different TAG, like ebook, tutorials, game, software etc. this project started four months only, i commit new change every day almost, thanks for your advice.

when the all function is over, i will make the donation page, but the function dev is first.

**Comment by JarHeadDev** [comment]
Dev functions(s) first makes sense to me. Looking forward to testing the continued progress on script. 

Is there a plan to have  a temp install/config like some scripts have in the future or will it stay as ssh?

**Comment by taobataoma** [comment]
@JarHeadDev maybe stay in ssh always, because the site start is over these ssh operation, if the site is not started, the install scripts can not running. do you have any good idea?

**Comment by dlmartinez1972** [comment]
wondering does the demo site update as you update the source here as well? 

I have no problem with using ssh. just some hosts it could be an issue. 

감사합니다

**Comment by taobataoma** [comment]
@dlmartinez1972 yes, the demo site is update from here direct by git pull.

**Comment by dlmartinez1972** [comment]
okay awesome to know. 

if you have time you should give TSSE and TSUE a look for ideas. The author has been MIA for a long time but it's one the top scripts still. 

We're currently running TSSE 7.5. I could setup a temp site/temp account if you like to get ideas or take screen shots. 

That might give you some feature ideas for the future. Not so much to clone it. The only thing I really hate about it is the licensing link back to the authors site and the tiny shoutbox text.

**Comment by dlmartinez1972** [comment]
I didn't want to recommend a bunch of features yet and waste your time because you've probably already got those things on the list of things to do.

I would say menu access control and access control by usergroup is the main thing I would recommend ideas about now though. It's important to keep non members eyes off menus they don't need access too like the admin menu.

In TSSE v7.5 it has in the admin panel a staff usergroup in case someone finds a way to the admin panel. They also back up the login to the admin panel with a pin also so makes it so have to know the password, pin and be in the admin staff usergroup list to do anything. 

Also, the staffmenu folder can be moved also to a non default path  www.mydomain.com/staffcp can be anything you want like www.mydomain.com/controlpanel

**Comment by NANO-Chile** [comment]
I was telling to @taobataoma exactly the same last night.

**Comment by taobataoma** [comment]
@dlmartinez1972 meanTorrent already realization menu access control, if no oper/admin logined, the admin menu will be hidden. the usergroup  is ['guest', 'user', 'oper', 'admin'], the admin can change it int user control manage center.

**Comment by NANO-Chile** [comment]
Dont forget about the VIP usergroup, is really important!

**Comment by dlmartinez1972** [comment]
agreed the VIP user group important. would be nice if can add custom user groups as well.

**Comment by dlmartinez1972** [comment]
most scripts have where if you add a section you set what user groups can view it. 

I wish someone make it so each user has and override option and checkboxes for each section because that would allow more control of content access. makes it would make it easier then having like 30+ user groups.

**Comment by NANO-Chile** [comment]
Totally agree! Custom user groups would be great, of course with custom permissions too.

**Comment by dlmartinez1972** [comment]
BTW I don't mine donating for your efforts. I know it takes alot of time to make software.

**Comment by NANO-Chile** [comment]
Same here, as I told you via Skype. Im really interested in your source, maybe someday I can move from TSSE to meanTorrent, I really wish that.

**Comment by dlmartinez1972** [comment]
same here. atm working on unzending TSSE before xam's site is gone for good.

**Comment by taobataoma** [comment]
@all, please give me some time to finished what are you wanted.

the user group in meanTorrent is fixed, meanTorrent used fashionable LEVEL to replace old user level, the LEVEL value is from user score, user can get score by any things, like stay in IRC, upload torrents, reply forum topic, seeding, downloading etc. the score system is almost finished now, I think this score level system is more flexible.

**Comment by dlmartinez1972** [comment]
be nice to have both user group options. 

personally I prefer the traditional user groups and you assign users to one. also custom user groups for limiting access by user group. 

I like the override idea also for allowing an admin to limit access to sections on a user by user option.
